system/cfme-setup.sh

```diff
@@ -31,6 +31,13 @@ cat <<'EOF' > /etc/httpd/conf.d/ssl.conf
 # upgraded.
 EOF
 
+# Incorporate the iptable updates
+if [[ -f /etc/sysconfig/iptables.miq ]]; then
```

Comment:
OOPS, just saw this now @jvlcek.  :tomato: 

I don't think this will ever be true. I think because we're no longer copying it.  If anything, we'd have to check `/var/www/miq/system/TEMPLATE/etc/sysconfig/iptables` right?